themes/default/content/blog/tf-migration-offer/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Terraform Migration Offer"
```

Comment:
Let's come up with a more exciting title. Even just saying "Announcing the ..." would spruce it up a bit.